libs/cardano-ledger-test/src/Test/Cardano/Ledger/Constrained/Lenses.hs

```diff
@@ -183,3 +202,23 @@ strictSeqListL = lens toList (\_ y -> fromList y)
 
 mapCompactFormCoinL :: Lens' (Map a (CompactForm Coin)) (Map a Coin)
 mapCompactFormCoinL = lens (Map.map fromCompact) (\_ y -> Map.map compactCoinOrError y)
+
+drepDelegationUMapL :: Lens' (UMap c) (Map (Credential 'Staking c) (DRep c))
+drepDelegationUMapL = lens dRepMap delta
+  where
+    delta um new = unSplitUMap (split {spDRep = new})
+      where
+        split = splitUMap um
```

Comment:
Looks identical to `drepUMapL` above ^
```suggestion
```

Reply:
Yeah, not sure how that happened. There is only one version now.